genestack_client/genestack_shell.py

```diff
@@ -298,14 +300,16 @@ def do_EOF(self, line):
 
     def process_command(self, command, argument_list, shell=False):
         """
-        Run a command with arguments.
+        Run a command with arguments, returns exit code.
```

Comment:
I don't think this appendix is worth mentioning, it is obvious.
`Runs the given command with the provided arguments`